requirements-dev-minimal.txt

```diff
@@ -1,3 +1,4 @@
 mypy
 pre-commit
 requirements-tools
+emoji
```

Comment:
The `emoji` package isn't necessary here, it's only needed in regular `requirements`.

The `requirements.txt` structure is usually unfamiliar to new OCF contributors, see the [requirements-tools docs](https://github.com/Yelp/requirements-tools#recommended-requirements-setup-for-applications) for an explanation of what each file means.

Reply:
Sorry this is my fault, I mis-remembered which file is the correct one.

Reply:
Ah this is my fault, I forgot it will use the regular requirements.txt

Reply:
This should also mean none of `requirements-dev.txt` needs to be updated, although minor package bumps there are probably fine anyway